File: backend/src/content/domain/reading_time.py
import re

_WORDS_PER_MINUTE = 200
_TAG_RE = re.compile(r"<[^>]+>")


def compute_reading_time(text: str) -> int:
    word_count = len(_TAG_RE.sub(" ", text).split())
    return max(1, round(word_count / _WORDS_PER_MINUTE))

File: mobile/src/extraction/readingTime.ts
const WORDS_PER_MINUTE = 200;

export function computeReadingTime(text: string): number {
  const wordCount = text.replace(/<[^>]+>/g, " ").split(/\s+/).filter(Boolean).length;
  return Math.max(1, Math.round(wordCount / WORDS_PER_MINUTE));
}
